stores/author.ts

```diff
@@ -0,0 +1,70 @@
+interface OwnedClassMapType {
+  classId: string
+  title: string
+}
+
+export const useAuthorStore = defineStore('AuthorStore', () => {
+  const ownedClassByWalletMap = ref<Record<string, NFTClass[]>>({})
```

Comment:
I think storing NFT class IDs is enough and add class data by `nftStore.addNFTClass()`
```suggestion
  const nftClassIdsByOwnerWalletMap = ref<Record<string, string[]>>({})
```

Reply:
I think the purpose was to allow filtering WNFT via `metadata?.['@type'] === 'Book'`